tests/common/experimental/test_tensor_collector_batch_size.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) 2023 Intel Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from abc import ABC
+from abc import abstractmethod
+from typing import List
+
+import numpy as np
+import pytest
+
+from nncf.common.graph.utils import get_reduction_axes
+from nncf.experimental.common.tensor_statistics.collectors import TensorCollector
+
+
+class TemplateTestTensorCollectorBatchSize(ABC):
+    @staticmethod
+    @abstractmethod
+    def get_tensor_statistics_class():
+        ...
```

Comment:
Can we use pass as we used to use before?

Reply:
done